libutils/path.c

```diff
@@ -0,0 +1,34 @@
+#include <path.h>
+#include <string_lib.h>
+
+const char *Path_Basename(const char *path)
```

Comment:
Might be useful to mention why the `basename()` function from *glibc* is not good.

Reply:
It's in the header.